geometry_msgs/PoseStamped object_pose
string class_name
float32 Register_value